internal/stacks/stack.go

```diff
@@ -38,11 +39,20 @@ type Member struct {
 	ExposedEthconnectPort   int    `json:"exposedEthconnectPort,omitempty"`
 	ExposedPostgresPort     int    `json:"exposedPostgresPort,omitempty"`
 	ExposedDataexchangePort int    `json:"exposedDataexchangePort,omitempty"`
-	ExposedIPFSApiPort      int    `json:"exposedIPFSApiPort,omitempty`
-	ExposedIPFSGWPort       int    `json:"exposedIPFSGWPort,omitempty`
+	ExposedIPFSApiPort      int    `json:"exposedIPFSApiPort,omitempty"`
+	ExposedIPFSGWPort       int    `json:"exposedIPFSGWPort,omitempty"`
```

Comment:
We should probably pull the linters over from `firefly` into this project to catch stuff like this. Outside the scope of this PR, but I'll probably open an issue to do that at some point soon.